Content.Shared/SSDIndicator/SSDIndicatorSystem.cs

```diff
@@ -9,10 +11,16 @@ public sealed class SSDIndicatorSystem : EntitySystem
 {
     public override void Initialize()
     {
+        SubscribeLocalEvent<SSDIndicatorComponent, ComponentInit>(OnInit);
         SubscribeLocalEvent<SSDIndicatorComponent, MindAddedMessage>(OnMindAdded);
         SubscribeLocalEvent<SSDIndicatorComponent, MindRemovedMessage>(OnMindRemoved);
     }
 
+    private void OnInit(EntityUid uid, SSDIndicatorComponent component, ComponentInit args)
+    {
+        component.IsSSD = HasComp<MindComponent>(uid);
```

Comment:
Is this necessary? It shouldn't be doing anything right now anyways since mind component is on mind entities, not on players. That would be MindContainerComponent.

Reply:
its also the opposite logic, ssd would be if there isnt a mind

Reply:
this is to prevent forcing ssd `true` when adding a component in runtime (and for npcs like dev mouse)
I added npc check in on init even